app/scripts/store/dimStores.directive.js

```diff
@@ -22,7 +22,7 @@
         "    'hide-filtered': vm.hideFilteredItems,",
         "    'show-elements': vm.showElements",
         '  }">',
-        '  <div dim-store-heading store-data="store"></div>',
+        '  <div dim-store-heading class="character dim-col-{{vm.charCol}}" store-data="store"></div>',
```

Comment:
When you say keep them all the same size, what does that mean?  Here's a 5x inventory and a 4x vault.  What's the same size here?  

![screen shot 2016-05-11 at 11 01 40 pm](https://cloud.githubusercontent.com/assets/984608/15203764/5ee0ca70-17cc-11e6-9a3f-c6ae70b8d6ed.png)

I think it should be full width of the vault column to keep it simple.